common/session/class.AnonymousSession.php

```diff
@@ -120,4 +120,10 @@ public function refresh()
     {
         // nothing to do here
     }
+
+    public function getContexts(string $class = null): array
+    {
+        return [];
```

Comment:
Why should we drop contexts support for anonymous?

Reply:
good idea, added support